src/main/app/pdf/issueReceipt.ts

```diff
@@ -19,6 +20,7 @@ export class IssueReceipt {
   data (): object {
     return {
       claim: ClaimMapper.createClaimDetails(this.claim),
+      totalClaimAmount: NumberFormatter.formatMoney(this.claim.totalAmountTillToday),
```

Comment:
I think this should be: `totalAmountTillDateOfIssue` @rlewan  @pragnesh-patel

Reply:
Yeah, you're right 🙈